packages/requests/src/lib/requests-result.ts

```diff
@@ -53,9 +56,9 @@ export interface IdleRequestResult extends BaseRequestResult {
   fetchStatus: 'idle';
 }
 
-export type RequestResult<TError = any> =
+export type RequestResult<TError = any, TData = any> =
```

Comment:
Add a test for the type, please

Reply:
`expectTypeOf(value).toEqualTypeOf<Type>();`

Reply:
Yes ^^